components/Buttons/src/MDCButton.m

```diff
@@ -124,6 +125,8 @@ @interface MDCButton () {
 }
 @property(nonatomic, strong) MDCInkView *inkView;
 @property(nonatomic, readonly, strong) MDCShadowLayer *layer;
+@property(nonatomic, assign, getter=ismdc_layerCornerRadiusAssigned)
```

Comment:
Given that this is a private API I think `layerCornerRadiusAssigned` and `isLayerCornerRadiusAssigned` are ok names.

Reply:
Done.